BlazorWasmDemo/Client/BlazorWasmDemo.Client.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>net6.0</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+    <LangVersion>11</LangVersion>
```

Comment:
Better remove this property from all the projects and update the one in the global Directory.Build.props instead.

Reply:
@iamcarbon Agreed, but could be separate PR.